Sources/Nitrozen-SwiftUI/Elements/Avatar/NitrozenAvatar.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  NitrozenAvatar.swift
+//  
+//
+//  Created by Keyur Patel on 18/04/23.
+//
+
+import SwiftUI
+
+public struct NitrozenAvatar: View {
+    
+    
+    //TODO: Hitendra - combine this common usage and change scope to align with Alert and actionSheet in future once other MR merged
+    public enum CustomImageView {
+        case systemImage(name: String) //customized image from SFSymbol
+        case assetImage(name: String) //customized image from .xcassets
+        case custom(view: AnyView) // whole customized view
+        case text(_ string: String) // for first 2 charector of text
+    }
+    
+    var mainView: CustomImageView
+    var isDisable: Bool
+    var appearance: NitrozenAppearance.Avatar
+    
+    private var maxCharacterCount: Int
```

Comment:
this is not customisable from out side as of now, right?
 so make it 
`private var initialsMaxCharacterCount: Int`